app/locale/en_US/Pay_Payment.csv

```diff
@@ -6,8 +6,9 @@
 "Percentage van totaalbedrag","Percentage of the total"
 "Om met billink te betalen, moet je akkoord gaan met de voorwaarden","To pay with billink you have to accept the terms and conditions"
 "Betaling geannuleerd","Payment canceled"
+"PAY. Betaalmethoden","PAY. Payment Methods"
```

Comment:
Ik zie dat je hier een regel toevoegd, gaat dit goede met de volgende regels die allemaal 1 verplaatsen?  
Waar is deze voor? @max-geraci

Reply:
Gaat goed met de volgende regels ja. Dit is de naam die links wordt weergegeven. We hadden hier altijd ons oude logo staan, nu is dat dus vervangen door deze tekst